cmd/podman/create.go

```diff
@@ -82,33 +92,27 @@ func createCmd(c *cli.Context) error {
 	return nil
 }
 
-func createInit(c *cli.Context) error {
+func createInit(c *cliconfig.PodmanCommand) error {
 	// TODO should allow user to create based off a directory on the host not just image
```

Comment:
We actually have support for this... If we're working in here right now, can you kill this first comment?